### PR TITLE
Add axis parameter to dice loss

### PR DESCRIPTION
**Pull request by td-jakubl**
https://github.com/keras-team/keras/issues/19637

### CODE REVIEW

**Comment by gbaned** [comment]
Hi @td-jakubl Can you please sign CLA. Thank you!

**Comment by td-jakubl** [comment]
> Hi @td-jakubl Can you please sign CLA. Thank you!

I did, but still showing: "Missing CLA from one or more contributors". I think the reason is that I did PR from company github account. Can we disable this PR and then I will do the same PR but from my private github account?

**Comment by td-jakubl** [comment]
> Hi @td-jakubl Can you please sign CLA. Thank you!

Done :)

**Comment by td-jakubl** [comment]
Hi @gbaned  some test fails and perhaps this is not related to changes in this PR. Do you have any idea why?

File: keras/src/losses/losses.py [hunk 1]
```diff
@@ -1961,28 +1961,42 @@ class Dice(LossFunctionWrapper):
 
     Returns:
         Dice loss value.
+
+    Examples:
+
+    y_true = tf.constant([[[[1.0], [1.0]], [[0.0], [0.0]]], [[[1.0], [1.0]], [[0.0], [0.0]]]])
```
Comment (fchollet) [inline keras/src/losses/losses.py:1967]: Please use a code block for the code example. Do not use TF APIs, only Keras APIs (backend agnostic)

**Review by fchollet — COMMENTED** [review]
Thanks for the PR! Please add a unit test.

File: keras/src/losses/losses.py (hunk 1 above)
Comment (td-jakubl) [reply keras/src/losses/losses.py]: Updated

**Review by fchollet — APPROVED** [review]
LGTM, thank you for the contribution!

**Comment by td-jakubl** [comment]
> LGTM, thank you for the contribution!

No, problem. I reformated code. Could you rerun workflows?

**Pull request merged**